examples/react-redux/js/reducers/todo.js

```diff
@@ -0,0 +1,48 @@
+import * as actionTypes from '../actions/actionTypes';
+
+const toggleTodo = state => {
+	return {
+		// Object spread operator is not supported by jshint yet
```

Comment:
Yes, `jshint ignore` works fine ignoring the spread operator. `jscs: disable` is the breaking one.